### ISSUE TITLE
GitHub/Linkedin links should have alt text

### ISSUE THREAD

**Issue by codeofdusk** [josephhutch/aether#63]
The profile links for GitHub, etc at the bottom of [the demo site](https://www.joehutch.com/posts/aether-theme/) don't have any alt text. I'm a screen reader user, so I need to click each link to see what it does.

**Comment by codeofdusk** [comment]
Closed in #64.
